app/lib/efile/dependent_eligibility/child_tax_credit.rb

```diff
@@ -30,13 +30,11 @@ def under_ctc_age_limit?
       private
 
       def is_qualifying_child?
```

Comment:
this line is ONLY to see if the client is a qualifying child, flat out. The line that was removed was intended to look at the ESD to see if they are a qualifying_child, which is sort of redundant since this logic will never run if we're looking at the qualifying attribute (qualifying_ctc) when it is an efile submission dependent object.